source/includes/opts/migrationName.rst

```diff
@@ -0,0 +1,18 @@
+.. reference/configuration.txt
+.. reference/mongosync.txt
+
+.. versionadded:: 1.7.0
+
+Sets a migration name for a sync operation. For example, you can set a
```

Comment:
I would remove the for example here, there are 3x "For examples" currently for this one parameter section.

```suggestion
Sets a migration name for a sync operation. You can set a
```

Reply:
Removed the other "for example" in the middle. That works a bit better, otherwise the user might think the migration name is the main reason for the name.